python/docs/lis/userguide.rst

```diff
@@ -56,26 +65,180 @@ also be reached directly from the Logical File:
 For a full overview of the content off all these records, please refer to :ref:`LIS Logical Records`.
 
 
-Read the curves
----------------
+Working with the curves
+-----------------------
+
+Getting the overview
+....................
 
 Within a LIS Logical File, curves are defined and organized by Data Format
 Specification Records (DFSR). A DFSR defines a set of channels/curves that are
 sampled along the same index. There might be multiple Data Format Specification
 Records in a Logical File, each defining different channels/curves and index.
 
-Read all the curve data by passing each DFSR to :func:`dlisio.lis.curves`:
+.. note::
+    LIS79 opens for the presence of duplicated DFSR, for redundancy. Currently,
+    dlisio has no support for identifying redundant DFSRs, and curves() will
+    return empty arrays for these.
+
+The :class:`dlisio.lis.DataFormatSpec` can be accessed directly from the
+:class:`dlisio.lis.LogicalFile`:
 
 .. code-block:: python
 
-    >>> for format_spec in f.data_format_specs:
-    ...    _ = lis.curves(f, format_spec)
+    >>> formatspecs = f.data_format_specs
+
+A DFRS contains information about the logset in general, such as logging
+direction and information about the index. In addition it contains a list of
+Spec Blocks, :attr:`dlisio.lis.DataFormatSpec.specs`. Each Spec Block describes
+one of the channels/curves in the logset. But let's begin by looking at the
+index:
+
+.. code-block:: python
+
+    >>> format_spec = formatspecs[0]
+    >>> format_spec.index_mnem
+    'TDEP'
```

Comment:
Maybe use DEPT here and below?
I guess you used some real file, but according to customer Subset, DEPT is dictionary-controlled value, while TDEP is not there.

(You also have inconsistencies in your testimony and use DEPT sometimes already :) )